yarn.lock

```diff
@@ -1829,14 +1829,9 @@ emoji-regex@^8.0.0:
   resolved "https://registry.yarnpkg.com/emoji-regex/-/emoji-regex-8.0.0.tgz#e818fd69ce5ccfcb404594f842963bf53164cc37"
   integrity sha512-MSjYzcWNOA0ewAHpz0MxpYFvwg6yjy1NG3xteoqz644VCo/RPgnr1/GGt+ic3iJTzQ8Eu3TdM14SawnVUmGE6A==
 
-emojis-list@^2.0.0:
-  version "2.1.0"
-  resolved "https://registry.npmjs.org/emojis-list/-/emojis-list-2.1.0.tgz#4daa4d9db00f9819880c79fa457ae5b09a1fd389"
-  integrity sha1-TapNnbAPmBmIDHn6RXrlsJof04k=
-
 emojis-list@^3.0.0:
   version "3.0.0"
-  resolved "https://registry.npmjs.org/emojis-list/-/emojis-list-3.0.0.tgz#5570662046ad29e2e916e71aae260abdff4f6a78"
+  resolved "https://registry.yarnpkg.com/emojis-list/-/emojis-list-3.0.0.tgz#5570662046ad29e2e916e71aae260abdff4f6a78"
```

Comment:
loader-utils@^1.0.2 is used by babel-loader I think. Probably has something to do with the tests.

Reply:
`babel-loader` support old versions of webpack, they drop it in future, ideally we should improve `getOptions` on webpack side like: `getOptions<T extends object>(schema: any): T;`

Reply:
It's definitely optional: https://github.com/webpack/webpack/blob/85fe6ac4f436384e2a20c7996c70f14e4172ccd8/lib/NormalModule.js#L495

It's our definition in this case that's causing the issue.  But I'm fine with that.  I'm hoping this will resolve it in future: https://github.com/webpack/webpack/pull/13164

Reply:
@alexander-akait: If I'm not mistaken you need `getOptions<T extends object>(schema?: any): T;` to make it truly optional in strict mode. That's why I had to send in `undefined`. I can be wrong here though.